docs/support.md

```diff
@@ -0,0 +1,48 @@
+# Reliability Kit support
+
+Reliability Kit is owned and maintained by the Customer Products Reliability team at the Financial Times.
+
+It's been designed to make our user-facing systems easier to support in production environments.
+
+However, we hope that it will be useful to other teams at the FT.
+
+## Releases and semantic versioning
+
+We follow [Semantic Versioning 2.0.0](https://semver.org/spec/v2.0.0.html).
+
+This means that we will not make breaking changes to Reliability Kit without a corresponding major
+release (like moving from `v1.0.0` to `v2.0.0`). These changes will be documented in a way that helps
+users migrate. We will consider the scale of these migrations and try to minimise the impact on the
+teams using this library and, where possible, we will mark features as deprecated some time ahead
+of a major release.
+
+## The support we provide
+
+### …within Customer Products
+
+The Reliability team is always available to work with teams in Customer Products. Our priority is making
+sure that FT.com, our iOS and Android apps and our Data Journalism work are available to our subscribers.
+
+Please contact us in [#cp-reliability on Slack](https://financialtimes.slack.com/archives/C02B89GEQHF) with
+any bug reports, feature ideas or requests for help.
+
+If you're experiencing an issue in production, message [#ft-next-support](https://financialtimes.slack.com/archives/C042NBBTM) and tag `@cp_reliability` so that we see it immediately.
+
+### …to other teams at the FT
+
+Where time allows, we would love to improve Reliability Kit so that it better meets the needs of other
+users at the FT. This could include:
+
+- reviewing minor code changes that will make these packages work better for you (please read the [contributing guide][] before opening pull requests)
```

Comment:
needs link

Reply:
It has one. It uses Markdown's [reference style links](https://www.markdownguide.org/basic-syntax/#reference-style-links), see the a couple of lines down for the actual link:

```markdown
[contributing guide]: https://github.com/Financial-Times/dotcom-reliability-kit/blob/main/docs/contributing.md
```

Though actually maybe it is better if we just use normal links as they're more widely known.

Reply:
although that guide doesn't cover what we're doing here, which is using a link with an empty reference. Still valid and works but even less widely known it seems 😅

This is valid Markdown and the text will link to the reference which contains the same value:

```markdown
Hello [this] is a link. [so is this][]

[this]: https://example.com/1
[so is this]: https://example.com/2
```